Added a stop command to just-bin-it (#25)
ess-dmsc/just-bin-it: README.md

This will start all the histograms counting from zero but will not change any other
settings, such as bin edges etc.

### Stoping counting
In order to stop counting send the stop command:
```json
{
  "cmd" : "stop"
}
```

> This will cause histogramming to stop and another, final, histogram to be sent. Restart with 
> either the `config` or ``restart` command.    

Reviewer: ```suggestion
This will cause histogramming to stop and the final histogram to be published.
```
Restart won't work this way any more as the process will be terminated on stop.
Perhaps it should be renamed to 'reset_count' or similar? What do you think?

Author: OK, May be it is a good idea to rename restart to reset_count then. And, yes, if restart does not do the trick, it has to go from the README in the stop section. 

I also have a mental model clash between Just-bin-it and how I think of histogramming. In my world, the Histogrammer is configured once and then told to start and stop histogramming. Where starting histogramming means clearing all the data and then histogramming and stop histogramming means, well, stopping histogramming and in the case of just-bin-it, send a final histogram. The rationale behind this is that configuration can mean transferring largish mapping arrays. Some of which can certainly be stored somewhere near just-bin-it as they rarely change. But some, like a time binning with user defined bins, may need to be transferred.

Reviewer: Thanks Mark. That is useful to know and I think this is something that would be achievable with a little work.
